src/status_im2/common/bottom_sheet/view.cljs

```diff
@@ -57,7 +58,8 @@
   [_ _]
   (let [sheet-height (reagent/atom 0)]
     (fn [{:keys [hide? insets theme]}
-         {:keys [content selected-item padding-bottom-override on-close shell?]}]
+         {:keys [content selected-item padding-bottom-override on-close shell?
+                 gradient-customization-color]}]
```

Comment:
better to just have `customization-color` imo. It will be shared across for anything used it, right?

Reply:
Updated 👍 Introduced a new prop `gradient-cover?` to toggle gradient cover in the bottom sheet.